specification/voiceservices/resource-manager/Microsoft.VoiceServices/cadl/voiceservices.cadl

```diff
@@ -0,0 +1,60 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-providerhub";
+import "@azure-tools/cadl-autorest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-azure-resource-manager";
+import "@cadl-lang/versioning";
+
+using Cadl.Http;
+using Cadl.Rest;
+using Cadl.Versioning;
+using Azure.ResourceManager;
+
+@armProviderNamespace("Microsoft.VoiceServices")
+@serviceTitle("Microsoft.VoiceServices")
+@serviceVersion("2022-12-01-preview")
+@versionedDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+namespace Microsoft.VoiceServices;
+
+// Various tooling enforces (or at least suggests) that all models should have a provisioning State field, as this means something special to ARM.
+
+@doc("Provisioning state of the resource.")
+enum ProvisioningStateEnum {
+  Succeeded, Failed, Canceled
+}
+
+@knownValues(ProvisioningStateEnum)
+model ProvisioningState  is string;
+
+@doc("General resource properties.")
+model ResourceProperties {
+	@doc("Resource provisioning state.")
+	@visibility("read")
+	provisioningState?: ProvisioningState;
```

Comment:
>provisioningState? [](http://example.com/codeflow?start=1&length=18)

Same for this, will provisioningState ever be missing from this property?

Reply:
As above, it's not required on a PUT, which is why we've made it optional. Is this incorrect?